ingress/controllers/nginx/nginx/command.go

```diff
@@ -128,7 +128,10 @@ func (ngx Manager) testTemplate(cfg []byte) error {
 		return err
 	}
 	defer tmpfile.Close()
-	defer os.Remove(tmpfile.Name())
+	// do not remove temporal nginx.conf file if log level is >= 3
+	if !glog.V(3) {
```

Comment:
hmm, why? this looks like a weird lateral dependency. are we just leaking the file to debug later?